shell.nix

```diff
@@ -0,0 +1,24 @@
+let
+  pkgs = import <nixpkgs> {};
+in
+pkgs.mkShell {
+  buildInputs = [
+    pkgs.checkstyle
+    pkgs.dive
+    pkgs.just
```

Comment:
This is a better `make` task runner. I put it here, since some of the dev tasks currently implemented as shell script, especially the simple "run-off" ones, could be implemented in a transparent way from a `justfile`.